src/test/java/uk/gov/pay/publicauth/it/PublicAuthResourceITest.java

```diff
@@ -38,6 +39,7 @@
 import static uk.gov.pay.publicauth.model.TokenSource.API;
 import static uk.gov.pay.publicauth.model.TokenSource.PRODUCTS;
 
+@SuppressWarnings("OptionalGetWithoutIsPresent")
```

Comment:
Usually we don’t like `@SuppressWarnings` on an entire class (too broad) but given this is a test class and `Optional.get()` is used in many places, I’ll let it past